Content.Shared/Mind/MindComponent.cs

```diff
@@ -116,4 +118,13 @@ public sealed partial class MindComponent : Component
     /// </summary>
     [DataField]
     public LocId? Subtype;
+
+
+    // sucks to save the information this way, find a better method
```

Comment:
Yeahhhhhhhhh I don't love it either. I can't think of anything better right away.

This and the field below should both have doc comments, though.